workers/unity/Packages/com.improbable.gdk.deploymentlauncher/DeploymentConfigurations.cs

```diff
@@ -97,15 +137,217 @@ public class SimulatedPlayerDeploymentConfig
         /// </summary>
         public string WorkerType;
 
-        internal SimulatedPlayerDeploymentConfig DeepCopy()
+        public SimulatedPlayerDeploymentConfig()
+        {
+            TargetDeploymentName = string.Empty;
+            FlagPrefix = string.Empty;
+            WorkerType = string.Empty;
+        }
+
+        internal new SimulatedPlayerDeploymentConfig DeepCopy()
         {
             return new SimulatedPlayerDeploymentConfig
             {
+                Name = Name,
+                SnapshotPath = SnapshotPath,
+                LaunchJson = LaunchJson,
+                Region = Region,
+                Tags = Tags.ToList(),
+
```

Comment:
new line

Reply:
This was intentional to separate the members of the base class and the derived class. Do you prefer them all together?

Reply:
don't have a strong preference in that case. we can leave it as it is